.postcssrc.yml

```diff
@@ -0,0 +1,3 @@
+plugins:
+  postcss-smart-import: {}
```

Comment:
what is this file for?

Reply:
11. postcss is configured out of the box together with cssnext and configured via `.postcssrc.yml`.